kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -180,4 +179,10 @@ object KyuubiSessionManager {
    * To keep compatibility, please do not change it.
    */
   val STATIC_BATCH_SECRET_UUID: UUID = UUID.fromString("c2ee5b97-3ea0-41fc-ac16-9bd708ed8f38")
+
+  /**
+   * The batch TProtocolVersion used for batch session handle.
+   * To keep compatibility, please do not change it.
+   */
+  val BATCH_PROTOCOL_VERSION: TProtocolVersion = TProtocolVersion.HIVE_CLI_SERVICE_PROTOCOL_V10
```

Comment:
unused

Reply:
It is a field of KyuubiBatchSession.

Reply:
you have defined a another field in `BatchesResource`
```scala
object BatchesResource {
  val REST_BATCH_PROTOCOL = TProtocolVersion.HIVE_CLI_SERVICE_PROTOCOL_V11
}
```

Reply:
ok, I see.